public/lang/babele/de/investigator.pathOfCthulhuAbilities.json

```diff
@@ -237,15 +237,15 @@
             "category": "Allgemein"
         },
         "Stealth": {
-            "name": "Verborgen bleiben",
+            "name": "Verstecken",
             "category": "Allgemein"
         },
         "Streetwise": {
-            "name": "Straßenwissen",
-            "category": "Sozial"
```

Comment:
This is an interesting one, because the "street" in "streetwise" isn't really about a type of road, it's a metaphor* for urban survival, in which case Gassen feels more correct. Is there a German equivalent that refers to the "urban survival" idea?

* Technically a synecdoche not a metaphor 🤓

Reply:
There is no direct translation, neiter for Streetwise nor for Urban Survical (=Überleben in der [Groß-]Stadt). 

As with many english terms you would have to use a whole sentence to get it 100% right. I believe Straßenwissen or Gassenwissen is the best we can do, but i prefer Gassenwissen für the started reason. 

The Term is from DTE (Das schwarz auge/The Dark Eye) the biggest RPG-System in Germany so ppl. know the concept and won't confuse it with navigational or architechtonical skills :)